### PR TITLE
chore(appengine): add support for application/json

### PR DESCRIPTION
**Pull request by jlsneto**
## Description
- allows you to create tasks for content-type application/json
- moved datetime import into the block [START cloud_tasks_appengine_create_task] (avoid import error for anyone reading the tutorial)

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: appengine/flexible/tasks/create_app_engine_queue_task.py [hunk 1]
```diff
@@ -44,6 +45,12 @@ def create_task(project, queue, location, payload=None, in_seconds=None):
             }
     }
     if payload is not None:
+        if isinstance(payload, dict):
+            # Convert dict to JSON string
+            payload = json.dumps(payload)
+            # specify http content-type to application/json
+            task["http_request"]["headers"] = {"Content-type": "application/json"}
+
```
Comment (jlsneto) [inline appengine/flexible/tasks/create_app_engine_queue_task.py:53]:
```suggestion
```

File: appengine/flexible/tasks/create_app_engine_queue_task.py [hunk 2]
```diff
@@ -15,14 +15,15 @@
...
 import argparse
-import datetime
 
 
 def create_task(project, queue, location, payload=None, in_seconds=None):
     # [START cloud_tasks_appengine_create_task]
     """Create a task for a given queue with an arbitrary payload."""
 
     from google.cloud import tasks_v2
+    import datetime
```
Comment (Onlineasistant) [inline appengine/flexible/tasks/create_app_engine_queue_task.py:25]: tarih saatini içe aktar

Comment (Onlineasistant) [inline appengine/flexible/tasks/create_app_engine_queue_task.py:25]: tarih saatini içe aktar

Comment (Onlineasistant) [inline appengine/flexible/tasks/create_app_engine_queue_task.py:25]: tarih saatini içe aktar

**Review by Onlineasistant — APPROVED** [review]

File: appengine/flexible/tasks/create_app_engine_queue_task.py [hunk 3]
```diff
@@ -15,14 +15,15 @@
...
 
 import argparse
-import datetime
 
 
 def create_task(project, queue, location, payload=None, in_seconds=None):
     # [START cloud_tasks_appengine_create_task]
     """Create a task for a given queue with an arbitrary payload."""
 
     from google.cloud import tasks_v2
```
Comment (Onlineasistant) [inline appengine/flexible/tasks/create_app_engine_queue_task.py:24]: `    ping = os.popen("/bin/ping -qc 1 "+host+" | awk -F/ '/^rtt/ { print $5 }'").read().strip()
`

File: appengine/flexible/tasks/create_app_engine_queue_task.py (hunk 2 above)
Comment (Onlineasistant) [inline appengine/flexible/tasks/create_app_engine_queue_task.py:25]: `    ping = os.popen("/bin/ping -qc 1 "+host+" | awk -F/ '/^rtt/ { print $5 }'").read().strip()
`

**Review by dinagraves — APPROVED** [review]

**Review by engelke — APPROVED** [review]
LGTM

**Review by parthea — APPROVED** [review]

**Pull request merged**